site/src/content/docs/roadmap.mdx

```diff
@@ -1,86 +1,75 @@
-- [ ] - Finalize and submit Zarf's `sandbox` application to officially join the OpenSSF.
-
-### Q3: Transfer Project, Stabilize and Extend
-
-- [ ] - Transfer project and additional repos (`zarf-ui`, `zarf-init-aws`, `setup-zarf`, etc.) to a new GitHub organization.
-- [ ] - Stabilize features after the consolidation of Q2 - clean up GA milestone in preparation for Q4.
-- [ ] - Flesh out the extension system for new features / experiments to be more smoothly integrated with Zarf.
-- [ ] - Make `zarf init` custom logic definable in-schema and across all packages.
-
-### Q4: Post Donation and General Availability
-
-- [ ] - Continue stabilizing features and interfaces in preparation for GA release.
-- [ ] - Gather and prepare to meet OpenSSF's `incubation` requirements.
-- [ ] - Officially cut a GA v1.0.0 release of Zarf.
+---
 
-## Feature Stability
+## 2025 Development Story
 
-Zarf has three levels of feature stability that are defined below but generally follow existing Kubernetes conventions. These communicate how a feature may change in the future as well as expectations for [\*ilities](https://en.wiktionary.org/wiki/ility) like documentation.
+### SDK Readiness (Q1 - Q2)
 
-### `alpha`
+We began the year with a deliberate push to lay the groundwork for SDK support. This involves:
 
-Alpha features act as early previews for things that are being considered for inclusion into Zarf and:
+- Refactoring internal logic to split the CLI from the shared library.
+- Establishing SDK entry points for Zarf functionality.
+- Introducing shared serialization and validation tooling to enable custom tooling built on Zarf.
 
-- they are very likely to have breaking changes (or be removed entirely) in future versions of Zarf
-- they will have limited documentation beyond autogenerated CLI and schema documentation (questions should be directed to the [Zarf team](https://kubernetes.slack.com/archives/C03B6BJAUJ3))
-- they will have limited testing and are only recommended for use cases where the risk of instability is acceptable
+Parallel to these efforts, we also focused on extending our test suite to cover decoupled components and ensure changes did not regress GA stability.
 
-### `beta`
+### Hardening & Observability (Q2 - Q3)
 
-Beta features are newer features that are being prepared to eventually become stable features and:
+As the architecture matured, attention turns to reliability and transparency:
 
-- they may introduce breaking changes before release, but will include any applicable workarounds
-- they will have documentation beyond autogenerated docs, but it won't be fully complete
-- they will have tests that cover their core features, but not covering every flavor of environment
+- Added in-depth error messages and diagnostic metadata to simplify user troubleshooting.
+- Hardened lifecycle to recover from partial failures or restarts.
+- Standardize on logging patterns for package lifecycle execution.
```

Comment:
The words added / hardened are in past tense while standardize is present tense. Also IMO for the Q2/Q3 I'd be worth mentioning Zarf values and namespace overrides. Maybe something like

- Provide alternative to Zarf variables to simplify user experience.
- Allow dynamically changes to the chart & manifest namespace on Deploy.